pom.xml

```diff
@@ -28,7 +28,7 @@
     </repository>
   </repositories>
   <dependencies>
-    <dependency>
+    <dependency> <!-- Junit dependency -->
```

Comment:
No need to add the XML comment here tbh.

Reply:
Only added as it seems to follow suit with the authors style of adding dependencies, can remove if you think it looks better.

Reply:
Maybe add JUnit dependency at the end of the list (where XML comments are not present)
I think imitating the prevailing style is a good thing but also think it makes sense to add a new dependency at the bottom.

Reply:
Good idea!